huggan/model_card_template.md

```diff
@@ -2,10 +2,10 @@
 tags:
 - huggan
 - gan
-task: image-generation
-license: 
-datasets:
-- 
+# See a lit of available tags here:
+# https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/Types.ts#L12
+# task: unconditional-image-generation or conditional-image-generation or image-to-image
```

Comment:
What's the difference between conditional image generation and image to image?

Reply:
Conditional image generation is generating an image conditioned on a class. image-to-image takes in an image and outputs an image (like animeGAN/arcane GAN)

Reply:
the inputs/outputs are different, that's all